MAUI/ProgressBar/Animation.md

```diff
@@ -0,0 +1,82 @@
+---
+layout: post
+title: Animation in .NET MAUI ProgressBar control | Syncfusion
+description: Learn here all about animation support in Syncfusion .NET MAUI ProgressBar control, its elements and more.
+platform: MAUI
+control: ProgressBar
+documentation: ug
+---
+
+# Animation in .NET MAUI ProgressBar (Progress Bar)
+
+The progress bar provides animation support to visualize the progress value changes in an interactive way. 
+
+The following properties are used to define the duration of animation for the specific states.
+
+* `AnimationDuration`: Represents animation duration of the determinate state’s progress indicator.
+* `SecondaryAnimationDuration`: Represents animation duration of the determinate state’s secondary progress indicator.
+* `IndeterminateAnimationDuration`: Represents animation duration of the indeterminate state’s indicator.
+
+## Easing effects
```

Comment:
Please add a gif output for this. Also, include a circular progress bar too.

Reply:
done